src/Core__List.resi

```diff
@@ -23,129 +23,158 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. */
 
 /**
-  Collection functions for manipulating the `list` data structures, a singly-linked list.
+Collection functions for manipulating the `list` data structures, a singly-linked list.
 
-  **Prefer Array** if you need any of the following:
+**Prefer Array** if you need any of the following:
 
-  - Random access of element
-  - Better interop with JavaScript
-  - Better memory usage & performance.
+- Random access of element
+- Better interop with JavaScript
+- Better memory usage & performance.
 */
 /** `'a t` is compatible with built-in `list` type */
 type t<'a> = list<'a>
 
 /**
-  Returns the length of a list.
+`length(list)` returns the length of `list`.
 
-  ```res example
-  List.length(list{1, 2, 3}) // 3
-  ```
+## Examples
+
+```rescript
+List.length(list{1, 2, 3}) // 3
+```
 */
 let length: t<'a> => int
 
-/** **See** [`length`](##length) */
+/**
+`size(list)`. **See** [`length`](##length)
```

Comment:
There are several comments using this format:

```
[`length`](##length)
```

In VSCode this is a link that opens a file that doesn't exist.

![image](https://user-images.githubusercontent.com/16160544/219235493-4003f220-8ee5-4c21-8db0-bb01af3d4d8a.png)


How should we reference functions?

Reply:
A proposal:

```
[`function`](ModuleName.function)
```

For function in same module:

```
[`function`](function)
```

Reply:
Right, that's an internal markdown link. We haven't 100% landed on the format yet (it's dependent on the documentation extraction, which we're working on in parallel). Let's revisit this a bit later, it's easy to fix once we have the format we want down.

Reply:
Ok